### ISSUE TITLE
Indentation warning for properties of argument object

### ISSUE THREAD

**Issue by JamesHenry** [typescript-eslint/typescript-eslint#11]
This issue was initially reported here: https://github.com/eslint/typescript-eslint-parser/issues/458

---

**What version of TypeScript are you using?**
2.7.1

**What version of `typescript-eslint-parser` are you using?**
14.0.0

**What code were you trying to parse?**

```ts
const registration = (
  state: RegistrationStoreState = initialState,
  action: {
    type: string;
    data: Object;
    error: Object;
  }
) => {
...
}
```

**What did you expect to happen?**
To see clear worskpace without warning

**What happened?**

I see warning `[eslint] Expected indentation of 2 spaces but found 4. (indent)`
And only when I format this code in this way:

```

const registration = (
  state: RegistrationStoreState = initialState,
  action: {
  type: string;
  data: Object;
  error: Object;
  }
) => {
...
}
```
I don't see warnings.

**Comment by armano2** [comment]
@bondom this issue is no longer present if you use `eslint-plugin-typescript` indent rule

------------

we have test scenario for this already
